nevergrad/functions/multiobjective/core.py

```diff
@@ -110,8 +110,9 @@ def pareto_front(self, size: Optional[int] = None, subset: str = "random") -> Li
             return random.sample([p[0] for p in self._points], size)
         possibilities: List[Any] = []
         scores : List[float] = []
-        for u in range(30):
-            possibilities += [random.sample(self._points, size)]
+        localrandom = random.Random(0)
```

Comment:
Seeding is never a good idea! random.Random() should work,
Once we store parameters, we can use the random number generator in it